### DISCUSSION TITLE
Feature Request: HTML

### DISCUSSION THREAD

**Discussion by Coskntkk** [katherinepeterson/readme.so#218] [Ideas]
Most people does not only use markdown, also uses html to create better readme files. It would be nice render not only markdown, also html. I will try to do it but, I think you can do it better, if you agree with me.

**Comment by octokatherine** [comment]
Interesting, I actually haven't seen that before! 
That feels maybe a little out of scope of this project, because that would be almost creating a website generator. I'm not sure how many people would use that feature either

**Reply by Coskntkk** [reply]
I made en example. You can check it out: https://coskntkk.github.io/viewme.md/

**Reply by octokatherine** [reply]
oh, you just mean render html on the screen? that should already be happening on readme.so if you click on the "Preview" tab for the right most section

**Reply by alvii147** [reply]
I think he's talking about entering raw HTML as Markdown and having that rendered. So for example, instead of doing `*...*` for italics, you should be able to do `<i>...</i>`. It's very common to leverage HTML on Markdown READMEs for cases where Markdown falls short (e.g. adjusting `img` size, setting custom anchored headers, [using collapsible sections](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections))

I can submit a PR for this, it's a simple fix.
